src/stable_browser.ts

```
@@ -443,8 +446,8 @@
             const element = elements[i];
             if (partial) {
               if (
-                (element.innerText && element.innerText.trim().includes(text)) ||
-                (element.value && element.value.includes(text))
+                (element.innerText && element.innerText.toLowerCase().trim().includes(text.toLowerCase())) ||
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)